files/default/sql/ddl/updates/1.3.0.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `hopsworks`.`conda_environment` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `tf_version` varchar(20) COLLATE latin1_general_cs NOT NULL,
+  `python_version` varchar(20) COLLATE latin1_general_cs NOT NULL,
+  UNIQUE KEY `tensorflow_python_version` (`tf_version`, `python_version`),
+  PRIMARY KEY (`id`)
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+ALTER TABLE `hopsworks`.`project` ADD  `conda_environment_id` int(11);
+ALTER TABLE `hopsworks`.`project` ADD FOREIGN KEY `anaconda_environment_fk` (`conda_environment_id`) REFERENCES `hopsworks`.`conda_environment` (`id`) ON DELETE SET NULL ON UPDATE NO ACTION;
```

Comment:
As I said above, I'm not convinced by the new table. But in any case, you should migrate existing environments to the new table structure.